hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -726,26 +722,19 @@
             <interface>10.10.1.*</interface>
         </interfaces>
         <ssl enabled="false">
-            <factory-class-name>com.hazelcast.nio.ssl.BasicSSLContextFactory</factory-class-name>
```

Comment:
What's the default for this? This is the _full_ example, so ideally all elements should be present with the default value. Not sure what to do if the default is empty though.

Reply:
The default value depends on the JVM version and netty+OpenSSL/BoringSSL on the classpath.
Users will usually touch the value if they want to switch explicitly from JSSE to OpenSSL/BoringSSL.

I'll think about updating the `<ssl>` section in the comment to be a real-world OpenSSL config.